test-configs.yaml

```diff
@@ -150,6 +150,15 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  am57xx-beagle-x15:
+    name: am57xx-beagle-x15
```

Comment:
The `name` and `dtb` values are derived from the config name by default when using the `arm-dtb` class, that is to say `am57xx-beagle-x15` in this case.  Since they all match, the `name` and `dtb` aren't strictly required here and can be dropped.

Reply:
Thanks, will drop.